core/instrumentation.ts

```diff
@@ -0,0 +1,16 @@
+import { OTLPHttpProtoTraceExporter, registerOTel } from '@vercel/otel';
+
+const host = process.env.LIGHTSTEP_COLLECTOR_HOST || 'ingest.lightstep.com';
+const port = process.env.LIGHTSTEP_COLLECTOR_PORT || '443';
+const otlpApiPath = process.env.LIGHTSTEP_API_PATH || 'v1/traces';
+
+const traceExporter = new OTLPHttpProtoTraceExporter({
+  url: `https://${host}:${port}/${otlpApiPath}`,
+  headers: {
+    'lightstep-access-token': process.env.LIGHTSTEP_ACCESS_TOKEN || '',
```

Comment:
I think this should be a specific Lightstep exporter since we have this hard-coded header